tests/Backend/CommonPart1/ImportExportCommonTest.php

```diff
@@ -489,10 +490,35 @@ public function testImportWithReorderedColumnsList(
     }
 
     /**
-     * @return \Generator<string, array{headersFile: string, importedFiles: array<string>, columns: array<string>, incremental: bool, expectedData: string, expectedColumns: array<string>}>
+     * @return \Generator<string, array{headersFile: string, importedFiles: array<string>, columns: array<string>, incremental: bool, withoutHeaders: bool, expectedData: string, expectedColumns: array<string>}>
      */
     public function tableImportReorderedData(): \Generator
     {
+        yield 'skip-headers' => [
+            'headersFile' => __DIR__ . '/../../_data/languages-headers.csv',
```

Comment:
Note to self: tohle vytvoří tabulku se sloupci id, name
https://github.com/keboola/storage-api-php-client/blob/a7e249b5549feca3c0652cfa5ae1923ac5d4ee3c/tests/_data/languages-headers.csv#L1